core/src/templates/components/card/card.twig

```diff
@@ -64,7 +64,7 @@
 
   {# Big ol button. Adds a BEM class for targeting. #}
   {%- if card_button_label is not empty %}
-    <div class="su-card__button">
+    <button class="su-card__button">
```

Comment:
I don't think this makes sense as we are using a `link` template inside.

Reply:
Yup, we can't change this one to `<button>` since there is a `<a>` nested inside (the link.twig). However, I wonder if we could get rid of the wrapper `<div class="su-card__button">` and just have the include with the link.twig? Perhaps the `div` was there for easier alignment?